### PR TITLE
Create terms-of-use.md

### PR DESCRIPTION
**Pull request by flexsurfer**
# Status Terms of Use

Last updated: 19 September 2024

### CODE REVIEW

File: terms-of-use.md
Comment (flexsurfer) [inline terms-of-use.md]: i'm not sure if `&nbsp;` is needed

**Comment by Samyoul** [comment]
Thank you @flexsurfer I'm waiting for Legal to give an answer to [this question about divergent features between Status app platforms.](https://discord.com/channels/1210237582470807632/1217171454940938341/1290657529612992544)

Also I can see the merge button on my side, so I don't think I've configured the repo correctly

**Comment by siddarthkay** [comment]
The ruleset for Legal Team approving all PRs was present but was inactive.
I've made it active now.

Screenshot 2024-10-08 at 10 20 55 AM

**Comment by flexsurfer** [comment]
thank you @siddarthkay now merge is not active as expected

**Comment by Samyoul** [comment]
Thank you very much @siddarthkay 🙏 🥳 .

Also I'm embarrassed that the rule was in place but it just needed turning on 💀 🤦

File: terms-of-use.md
Comment (Samyoul) [reply terms-of-use.md]: Probably not, I think it would be fine to remove all `&nbsp`s. As long as the files are identical in all repos then it will be fine.

**Comment by Samyoul** [comment]
Resolves https://github.com/status-im/status-software-legal-documents/issues/2

**Review by ericnguyen101 — APPROVED** [review]

**Review by Samyoul — APPROVED** [review]
Thank you @flexsurfer . Would you be able to merge this PR now?

**Pull request merged**
